_includes/translate-link

```diff
@@ -0,0 +1,3 @@
+<a href="{{ include.href_base | default: "" }}{% include translate text=include.href language=include.language %}">
```

Comment:
I think links should not be handled differently than a normal text because URLs are nothing else than text. What do you think @niccokunzmann ?